tests/unit/parts/test_yaml_utils.py

```diff
@@ -0,0 +1,89 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2022 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import io
+from textwrap import dedent
+
+import pytest
+
+from snapcraft import errors
+from snapcraft.parts import yaml_utils
+
+
+def test_yaml_load():
+    assert (
+        yaml_utils.load(
+            io.StringIO(
+                dedent(
+                    """\
+        entry:
+            sub-entry:
+              - list1
+              - list2
+        scalar: scalar-value
+    """
```

Comment:
This is just nitpicking, but indents here look ugly/strange (I usually align the quotes and dedented content).

Reply:
black did this :see_no_evil: